src/manifold/src/properties.cpp

```diff
@@ -201,7 +201,7 @@ struct CheckHalfedges {
 
     const Halfedge paired = halfedges[halfedge.pairedHalfedge];
     bool good = true;
-    good &= paired.pairedHalfedge == edge;
+    good &= paired.pairedHalfedge == static_cast<int>(edge);
```

Comment:
Will the update of `Halfedge` happen with `Collider`?

Reply:
Yes.